src/main/java/com/example/routtracker/models/Trip.java

```diff
@@ -0,0 +1,12 @@
+package com.example.routtracker.models;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class Trip {
+    private String originAddress;
+    private String destinationAddress;
+    private double amount;
```

Comment:
Java posiada dedykowaną klasę `Money` do przechowywania pieniędzy, ale na początek możesz użyć  `BigDecimal`, aby zbytnio nie komplikować.

https://www.baeldung.com/java-money-and-currency